src/main/java/de/hysky/skyblocker/config/configs/DungeonsConfig.java

```diff
@@ -39,6 +39,9 @@ public class DungeonsConfig {
     @SerialEntry
     public Livid livid = new Livid();
 
+    @SerialEntry
+    public Goldor goldor = new Goldor();
+
```

Comment:
This needs to go after `devices` to keep the order consistent (for merge conflict purposes).